### PR TITLE
Added release notes for GUI 0.20.0

### PR DESCRIPTION
**Pull request by gep13**
## Description Of Changes

Added in the release notes for Chocolatey GUI 0.20.0

## Motivation and Context

This is shipping soon, and we need the release notes to be there ready for notifications.

## Testing

1. Ran the .\preview.ps1 and looked at the output

## Change Types Made

* [ ] Bug fix (non-breaking change)
* [ ] Feature / Enhancement (non-breaking change)
* [ ] Breaking change (fix or feature that could cause existing functionality to change)
* [ ] PowerShell code changes.

## Related Issue

N/A

## Change Checklist

* [x] Requires a change to the documentation
* [x] Documentation has been updated
* [ ] Tests to cover my changes, have been added
* [ ] All new and existing tests passed.
* [ ] PowerShell v2 compatibility checked.

### CODE REVIEW

File: input/en-us/chocolatey-gui/release-notes.md [hunk 1]
```diff
@@ -12,6 +12,47 @@ This covers changes for the "chocolateygui" package, which is available as FOSS.
 
 > :memo: **NOTE** For commercial editions, please also refer to [Licensed Release Notes](xref:licensed-extension-release-notes), as well at the [Chocolatey GUI Licensed Extension Release Notes](xref:chocolatey-gui-licensed-extension-release-notes).
 
+## [0.20.0]() (February 10, 2022)
```
Comment (pauby) [inline input/en-us/chocolatey-gui/release-notes.md:15]: You adding the milestone link in later?

File: input/en-us/chocolatey-gui/release-notes.md [hunk 2]
```diff
@@ -12,6 +12,47 @@ This covers changes for the "chocolateygui" package, which is available as FOSS.
...
+
+> :warning: **WARNING**
+>
+> If you use Chocolatey GUI alongside the Chocolatey GUI Licensed Extension, if you install v0.29.0 of Chocolatey GUI then you will also have to install v0.4.0 of Chocolatey GUI Licensed Extension, or you will see errors when attempting to run Chocolatey GUI.
```
Comment (pauby) [inline input/en-us/chocolatey-gui/release-notes.md:19]:
```suggestion
> If you install v0.20.0 of Chocolatey GUI then you will also have to install v0.4.0 of Chocolatey GUI Licensed Extension if you use it. You will see errors when attempting to run Chocolatey GUI if you do not do so.
```

**Review by pauby — CHANGES_REQUESTED** [review]

File: input/en-us/chocolatey-gui/release-notes.md (hunk 1 above)
Comment (gep13) [reply input/en-us/chocolatey-gui/release-notes.md:15]: Indeed!

**Comment by gep13** [comment]
@pauby I have fixed up the requested changes, can you take another look?

**Review by pauby — APPROVED** [review]

**Pull request merged**